typescript/infra/scripts/helloworld/kathy.ts

```diff
@@ -57,6 +89,32 @@ async function sendMessage(
   console.log(JSON.stringify(receipt.events || receipt.logs));
 }
 
+function getPushGateway(): Pushgateway | null {
```

Comment:
Can we de-dup some of this and `submitMetrics` with the code in the relayer funder, i.e. make them available as common utils or something?
While we're at it, it'd be nice to also use the structured logging here that we do in the relayer funder. It'd make it a lot nicer to browse these logs when they're deployed